### ISSUE TITLE
Missing region when deploying function with --prune option

### ISSUE THREAD

**Issue by HyperBrain** [arabold/serverless-autoprune-plugin#2]
### Reproduction

(1) serverless function deploy _function name_ --prune

Leads to:
`Missing region in config` exception in

```
doCall (****\node_modules\serverless-autoprune-plugin\lib\AutoPrunePlugin.js:19:5)
```
### Expected behavior

Plugin should use the `evt.options.stage` and `evt.options.region` options set either by command line or by prompt.

**Comment by tobyhede** [comment]
Have the same issue.

**Comment by arabold** [comment]
Thanks for the bug report. I've just published version 0.2.1 which should support exactly this case.

`sls function deploy <Function Name> --prune` will prune the deployed function. It chooses the same stage and region as specified for `function deploy`. However, be aware the typically the function names for stages are identical and thus it doesn't matter whether you deploy to `dev`, `test`, or `prod` - it's all the same Lambda.

`sls function prune --all` will automatically prune all Lambdas in all deployed regions.

Let me know if this works.
